### PR TITLE
Remove EMBER_CLI_FASTBOOT environment variable and not include fastbo…

### PR DESCRIPTION
**Pull request by tsubomii**
This change makes ember-network work with ember-cli-fastboot 1.0.  It's backwards compatible. Works with ember-cli-fastboot that generates double build and without render using fastboot.

### CODE REVIEW

**Comment by kratiahuja** [comment]
I think `ember-network` has no fastboot tests. It would be nice to have some tests in here to make sure it works in CI too.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-network",
-  "version": "0.3.1",
+  "version": "0.3.2",
```
Comment (kellyselden) [inline package.json]: The version bump should be independent of this PR

**Review by kellyselden — CHANGES_REQUESTED** [review]
Version issue

**Comment by kellyselden** [comment]
I got `BroccoliMergeTrees: Expected Broccoli node, got undefined for inputNodes[0]` when trying this branch out.

**Comment by kratiahuja** [comment]
@kellyselden these changes are same as `ember-fetch` which worked for me. Did you try it in a dummy app?

File: package.json (hunk 1 above)
Comment (tomdale) [reply package.json]: @kellyselden is right, would you mind removing this from the PR? We will bump the version in the `package.json` when we publish to npm.

**Comment by tsubomii** [comment]
@kellyselden  I tried your app. The build works after I upgrade to ember-cli 2.14.0-beta.2, that’s the compatible version with ember-cli-fastboot 1.0 .

**Comment by tomdale** [comment]
Thanks @tsubomii!

**Pull request merged**